config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-config.yaml

```diff
@@ -624,8 +624,8 @@ presubmits:
               value: "true"
             - name: TEST_WINDOWS
               value: "true"
-            - name: WINDOWS_FLAVOR
```

Comment:
I think original value `containerd-2022` is ignored, it's actually running on 2019, capz windows 2022 cluster is quite unstable now

Reply:
@marosset @jsturtevant do you have any thoughts here?

Reply:
My thoughts are
- if the job has 2022 in the name we should update them to run windows server 2022 even if there is instability
- If the jobs specify an OS we should remove the invalid `WINDOWS_FLAVOR` env var to avoid confusion

I see a lot of these jobs do not specify the OS in the job name but I do see some that do

Reply:
That makes sense. If various CSI stories on top of windows 2022 are unstable we should demote those tests to ad hoc only, or delete them entirely.

Reply:
I think we could just remove following config since it's actually setting up 2019 windows cluster.
```
            - name: WINDOWS_FLAVOR
              value: "containerd-2022"
```